tox.ini

```diff
@@ -4,9 +4,10 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py{36,37,38,39,310}
+envlist = py{37,38,39,310,311}
 
 [testenv]
-deps = pytest
-   pytest-cov
-commands = py.test
+deps =
+    pytest
+    pytest-cov
+commands = pytest
```

Comment:
Similarly to the above comment about Travis, it's not clear to me that `tox` is used anymore either.